JitsiConference.js

```diff
@@ -1738,14 +1747,15 @@ JitsiConference.prototype.isCallstatsEnabled = function() {
 JitsiConference.prototype._onTrackAttach = function(track, container) {
     const isLocal = track.isLocal();
     let ssrc = null;
+    const isP2P = track.isP2P;
+    const remoteUserId = isP2P ? track.getParticipantId() : 'jitsi';
+    const peerConnection
+        = isP2P
+            ? this.p2pJingleSession && this.p2pJingleSession.peerconnection
```

Comment:
There is a getter, but it's based on the conference state rather than the flag set on the remote track. I think it's more clear this way and more "safe", but eventually I can change it if you really insist (I'd rather keep it this way though).